powerquery-docs/connectors/databricks.md

```diff
@@ -84,3 +87,24 @@ To connect to Databricks from Power Query Online, take the following steps:
 4. Once you successfully connect, the **Navigator** appears and displays the data available on the server. Select your data in the navigator. Then select **Next** to transform the data in Power Query.
 
     :::image type="content" source="./media/databricks/pq-choose-data.png" alt-text="Image of Power Query navigator loading Databricks Cloud data to online app.":::
+
+## Arrow Database Connectivity driver connector implementation
+
+You can use the [Arrow Database Connectivity (ADBC)](https://github.com/apache/arrow-adbc/blob/main/csharp/src/Drivers/BigQuery/readme.md) driver for the Databricks connector in Power BI. As we continue to enhance and add new capabilities to this connector, we recommend you to upgrade to the latest version to try it out and [submit product feedback](https://docs.databricks.com/aws/en/resources/ideas).
```

Comment:
In the second sentence, change "...we recommend you to upgrade..." to "...we recommend that you upgrade...".